cartoframes/data/observatory/enrichment/enrichment.py

```diff
@@ -80,16 +83,10 @@ def enrich_points(self, dataframe, variables, geom_col=None, filters={}):
             ...     geom_col='the_geom')
 
         """
-        variables = prepare_variables(variables, self.credentials)
-        geodataframe = self._prepare_data(dataframe, geom_col)
-
-        temp_table_name = self._get_temp_table_name()
-        self._upload_data(temp_table_name, geodataframe)
-
-        queries = self._get_points_enrichment_sql(temp_table_name, variables, filters)
-        return self._execute_enrichment(queries, geodataframe)
+        dataframe_enriched = self._enrich(GEOM_TYPE_POINTS, dataframe, variables, geom_col, filters or {})
```

Comment:
Better `return self._enrich(...`

Reply:
This is not needed: `or {}`

Reply:
> This is not needed: or {}

Yes, we need it to ensure a dict is sent to enrichment service

Reply:
The `_enrich` method contains:

```py
if filters is None:
    filters = {}
```

So `filters=None` is valid because it generates a `{}` inside the function.

Reply:
You are right